sonarqube/content.md

```diff
@@ -55,16 +55,55 @@ By default, the image will use an embedded H2 database that is not suited for pr
 
 Setup a database by following the "Installing the Database" section of https://docs.sonarqube.org/latest/setup/install-server/.
 
-### Use bind-mounted folders
+### Use bind-mounted folders or volumes
 
-The images contain the SonarQube installation at `/opt/sonarqube`. You need to use bind-mounted folders to override selected files or directories :
+The images contain the SonarQube installation folders at `/opt/sonarqube`. You will need to bind them to your host to override selected files or directories :
 
 -	`/opt/sonarqube/conf`: configuration files, such as `sonar.properties`
 -	`/opt/sonarqube/data`: data files, such as the embedded H2 database and Elasticsearch indexes
 -	`/opt/sonarqube/logs`: contains SonarQube logs about access, web process, CE process, Elasticsearch logs
--	`/opt/sonarqube/extensions`: plugins, such as language analyzers
+-	`/opt/sonarqube/extensions`: plugins, such as language analyzers  
```

Comment:
This extra space looks like it was probably just a minor misclick:

```suggestion
-	`/opt/sonarqube/extensions`: plugins, such as language analyzers
```

(Otherwise this all looks good :+1:)